fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -936,7 +933,11 @@ public Partition dropPartition(long dbId, String partitionName, boolean isForceD
     // get partition by name, not including temp partitions
     @Override
     public Partition getPartition(String partitionName) {
```

Comment:
Why change the behavior of this method?
The comment of this method show that it should not include temp partition

Reply:
BindSink. bindPartitionIds will convert partition name to partition id by calling this method，`OlapTable.getPartition(long partitionId)` has the same change , now I change the comment of two method